.github/PULL_REQUEST_TEMPLATE/rfc.md

```diff
@@ -0,0 +1,32 @@
+---
+name: RFC Template
+about: A design RFC for Zebra
+title: ''
+labels: C-design
+assignees: ''
+
+---
+
+<!--
+This template is for design RFCs.  The RFC process is documented in CONTRIBUTING.md
+
+- [ ] Copy the fields from the text of your RFC below
+- [ ] Leave the Design PR number blank, then fill it in *after* you create the PR (and the issue number is created)
+- [ ] After creating the Design PR, take the next available RFC number (not conflicting with any lower-number RFCs) and name your RFC book/rfcs/XYZT-my-rfc.md).
```

Comment:
Name the PR name? Or the filename of the .md ?

Reply:
Should be the filename of the .md, I'll clarify.  Actually this was correctly described in the Contributing file so I think your comment below is a good concern.  Maybe I should edit this to just refer to that description?

Reply:
Edited to just refer to the description.